frontend/packages/kubevirt-plugin/integration-tests/tests/vm.resources.scenario.ts

```diff
@@ -123,6 +124,16 @@ describe('Test network type presets and options', () => {
   const wizard = new Wizard();
   const leakedResources = new Set<string>();
 
+  const fillGeneralStep = async () => {
```

Comment:
we could go a bit further. Can we split the processWizard logic into the steps and call processGeneralStep here instead?

we can also use getBasicVMBuilder() or one of the presets here

Reply:
Updated.

Reply:
Great, thanks!